src/test/java/de/caritas/cob/userservice/api/adapters/web/controller/UserControllerChatE2EIT.java

```diff
@@ -182,6 +185,7 @@ class UserControllerChatE2EIT {
 
   @AfterEach
   void reset() {
+    cacheManagerConfig.cacheManager().getCache("rocketChatUserCache").clear();
```

Comment:
I'm fine with this @Leandro13Silva13 do we have this also in other repos where this cache is active?